modules/ossm-rn-new-features.adoc

```diff
@@ -134,8 +134,19 @@ This release marks the end of support for {SMProductShortName} Control Planes ba
 * Gateway injection is not a supported feature.
 
 === Kubernetes Gateway API
-Kubernetes Gateway API is a technology preview feature that is disabled by default.
+Kubernetes Gateway API is a technology preview feature that is disabled by default. If the Kubernetes API deployment controller is disabled, you must manually deploy and link an ingress gateway to the created Gateway object.
 
+If the Kubernetes API deployment controller is enabled, then an ingress gateway automatically deploys when a Gateway object is created.
+
+==== Installing the Gateway API CRDs
+The Gateway API CRDs do not come pre-installed by default on OpenShift clusters. Install the CRDs prior to enabling Gateway API support in the SMCP.
+
+[source,terminal]
+----
+kubectl get crd gateways.gateway.networking.k8s.io || { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.0" | kubectl apply -f -; }
```

Comment:
```suggestion
$ kubectl get crd gateways.gateway.networking.k8s.io || { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.0" | kubectl apply -f -; }
```
For all CLI command samples in the OCP docs, we need to add either `$` for regular user or `#` for root privileges to indicate the required permission level of the user issuing this command. Not sure what you need here, but it doesn't look like this requires root. See https://redhat-documentation.github.io/supplementary-style-guide/#commands-with-root-privileges

Reply:
Good point, @bburt-rh. I've made the edit and updated the PR. After the preview build completes, I'll confirm the change and move the story to ready to release. Thanks!

Reply:
change made, revised PR and preview build available.